api/clients/accountant.go

```diff
@@ -160,6 +159,37 @@ func (a *accountant) GetRelativeBinRecord(index uint32) *BinRecord {
 	return &a.binRecords[relativeIndex]
 }
 
+func (a *accountant) SetPaymentState(paymentState *disperser_rpc.GetPaymentStateReply) {
+	a.minNumSymbols = uint32(paymentState.PaymentGlobalParams.MinNumSymbols)
```

Comment:
these will panic if any of the attributes are nil?

Reply:
yes.... added a check for all of them!
It would've been great if protobuf let me set fields to be required, but proto3 removed "required" keyword:(

Reply:
If you use getters (i.e. `GetXXX`), you could chain them!